FirstTask.py

```diff
@@ -0,0 +1,180 @@
+from os import system as sys
+
+class BankAccount:
+    def __init__(self, Name, PassCode, AccNumber, Balance, AccoutType):
+        self.Name = Name
+        self.Passcode = PassCode
+        self.Balance = Balance
+        self.AccountType = AccoutType
+        self.AccNumber = AccNumber
+
+    def FullName(self):
+        return self.Name
+
+    def AccountNumber(self):
+        return self.AccNumber
+
+    def PassCode(self):
+        return self.Passcode
+
+    def CheckBalance(self):
+        return self.Balance
+
+    def CheckAccountType(self):
+        return self.AccountType
+
+    def AccDetails(self):
+        print("Account # ", self.AccNumber)
+        print("Name: ", self.Name)
+        print("Balance: ", self.Balance)
+        print("Account Type: ", self.AccountType)
+
+    def DepositBill(self,Bill):
+        if Bill >= 0:
+            self.Balance += Bill
+            return True
+        else:
+            return False
+
+    def DrawMoney(self,Money):

```

Comment:
Same here. A space after the comma, name should be `draw_money` and param should be `money`.